AEPMessaging/Sources/UI/ContentCards/ContentCardUI.swift

```diff
@@ -43,6 +43,11 @@ public class ContentCardUI: Identifiable {
         proposition.items.first?.contentCardSchemaData?.meta
     }
 
+    /// Method to track custom interaction on the content card
+    public func trackInteraction(_ interaction: String) {
+        proposition.items.first?.contentCardSchemaData?.track(interaction, withEdgeEventType: .interact)
```

Comment:
i think we should log something in the event any of these optionals is nil.

Reply:
Gotcha! Will update it